### PR TITLE
Start a gRPC server for E2 interface per tower

### PR DESCRIPTION
**Pull request by SeanCondon**
Only the UEs attached to that tower should be active for that server

Traffic sim Port is still at 5150, but the E2 ports start from 5152

* For each server started put an entry in to `onos-topo`
* For each server create an additional server port on the Kubernetes API

### CODE REVIEW

File: pkg/northbound/e2/e2.go [hunk 1]
```diff
@@ -18,27 +18,53 @@ import (
...
 	"google.golang.org/grpc"
 )
 
 var log = liblog.GetLogger("northbound", "e2")
 
-// NewService returns a new trafficsim Service
-func NewService() (service.Service, error) {
-	return &Service{}, nil
+// NewTowerServer - start a new gRPC server per tower
+func NewTowerServer(towerIndex int, serverParams utils.ServerParams) error {
```
Comment (adibrastegarnia) [inline pkg/northbound/e2/e2.go]: The idea is to have one separate connection from onos-ric to each base station? I mean there is no proxy thing involved correct?

Comment (SeanCondon) [reply pkg/northbound/e2/e2.go]: that is correct - each base station is listed with its own endpoint address in onos-topo

Comment (SeanCondon) [reply pkg/northbound/e2/e2.go]: I've tested it up to 1024 base station and have not seen any problems

Comment (adibrastegarnia) [reply pkg/northbound/e2/e2.go]: OK. Good. So for now, we deal with 1:1 connections.

**Pull request merged**